### ISSUE TITLE
Compatibility with Autoprefixer

### ISSUE THREAD

**Issue by voltronik** [10up/flexibility#27]
Using flexibility.js in combination with Autoprefixer (built into bdkjones/CodeKit) causes the `-js-display: flex;` property to be expanded to read: 

```
-js-display: -webkit-box;
-js-display: -webkit-flex;
-js-display: -moz-box;
-js-display: -ms-flexbox;
-js-display: flex;
```

Any thoughts on how to make this 'neater'?

**Comment by jonathantneal** [comment]
I’m unsure why Autoprefixer would touch an unknown or invalid prefix, so I’ve forwarded this issue to Autoprefixer.

**Comment by jonathantneal** [comment]
@voltronik, according to the folks at Autoprefixer, this was a bug in their software that was fixed in 6.3.1. You’ll need to update. See: https://github.com/postcss/autoprefixer/issues/608

**Comment by voltronik** [comment]
Brill, thanks. I'll wait for an update to CodeKit.
